### PR TITLE
BD-4083 - React Native Min App upgrade to 0.69 and link to rn 0.69 plugin

### PR DESCRIPTION
**Pull request by datkinnguyen**
Tickets:
- Plugin: https://bluedotinnovation.atlassian.net/browse/BD-3990
- Min App: https://bluedotinnovation.atlassian.net/browse/BD-4083

Test by pointing to the 0.69.4 plugin by changing Min App's package.json dependency to:
`"bluedot-react-native": "Bluedot-Innovation/Bluedot-React-Native-Plugin#dn/react-native-0.69",`

Once done testing + publishing our plugin, will need to reset it back to the official Bluedot-React-Native-Plugin

Most of the file changes are due to re-creating the project with new expo, react & react-native. Some manual changes are commented on for easier review.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,24 +10,24 @@
...
-    "react-native-permissions": "^3.3.1",
-    "react-native-push-notification": "^8.1.1",
-    "react-native-reanimated": "~2.3.1",
-    "react-native-screens": "~3.10.1",
-    "react-native-web": "0.17.1",
-    "react-router-dom": "^6.2.2",
-    "react-router-native": "^6.2.2"
+    "appcenter": "4.4.5",
+    "appcenter-crashes": "4.4.5",
+    "bluedot-react-native": "Bluedot-Innovation/Bluedot-React-Native-Plugin#dn/react-native-0.69",
```
Comment (datkinnguyen) [inline package.json:17]: pointing to 0.69 branch of plugin repo for QA. Will point to official version 2.1.3 when it's done QA and published

File: README.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
 # Bluedot-React-Native-Minimal-Integration
 
-This App is a minimal App depicting integration of Bluedot Plugin(bluedot-react-native) https://www.npmjs.com/package/bluedot-react-native
-with a React Native App supporting react native version 0.67.3
+This App is a minimal App depicting integration of Bluedot Plugin(bluedot-react-native) https://www.npmjs.com/package/bluedot-react-native with a React Native App supporting react native version 0.69.4
```
Comment (datkinnguyen) [inline README.md:3]: Changed in this file

File: ReleaseNotes.md [hunk 3]
```diff
@@ -1,5 +1,4 @@
-# Bluedot ReactNative Minimal Integration v2.1.2
+# Bluedot ReactNative Minimal Integration v2.1.3
 
-## New changes
-* Using Bluedot plugin 2.1.2 (support react-native 0.67.x)
-* Recreated the project using Expo SDK 44 and react-native 0.67.3
\ No newline at end of file
+- Using [Bluedot plugin 2.1.3](https://github.com/Bluedot-Innovation/Bluedot-React-Native-Plugin) (support react-native 0.69.4)
+- Recreated the project with react 18.0.0, react-native 0.69.4
```
Comment (datkinnguyen) [inline ReleaseNotes.md:4]: Changed in this file

**Review by duclong201 — APPROVED** [review]

**Review by nehabluedot — APPROVED** [review]
Good to go!!

**Pull request merged**